### PR TITLE
Adds workflow to publish to github pages

### PR DESCRIPTION
**Pull request by c-git**
Code is published to the gh-pages branch of the same repo.
GitHub Pages will need to be enabled and the custom domain [setup](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages) and verified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ the Hayden at his personal email address
 ## Disclaimer
 
 Rust for Lunch is not associated with or endorsed by either the
```
Comment (hds) [inline README.md]: If we use "nor", then it should be preceeded by "neither" (not either).

```suggestion
Rust for Lunch is not associated with or endorsed by neither the
```

File: content/about.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ the Hayden at his personal email address
 ## Disclaimer
 
 Rust for Lunch is not associated with or endorsed by either the
```
Comment (hds) [inline content/about.md]: If we use "nor", then it should be preceeded by "neither" (not either).

```suggestion
Rust for Lunch is not associated with or endorsed by neither the
```

**Review by hds — CHANGES_REQUESTED** [review]
Thanks for making this work.

Just a small bit of grammar stuff.

File: content/about.md (hunk 2 above)
Comment (c-git) [reply content/about.md]: Right you are, my brain read that part "correctly" but when it got to the "or" it was like nope.

**Comment by c-git** [comment]
No problem you're most welcome, I just copied my setup and modified as needed.

**Review by hds — APPROVED** [review]
Looks good. Thanks for the workflow!

**Pull request merged**